packages/opal-server/opal_server/worker.py

```diff
@@ -42,6 +42,13 @@ async def delete_scope(self, scope_id: str):
         scope_dir = self._base_dir / "scopes" / scope_id
         shutil.rmtree(scope_dir, ignore_errors=True)
 
+    async def periodic_check(self):
+        scopes = await self._scopes.all()
+
+        for scope in scopes:
+            if scope.policy.poll_updates:
+                sync_scope.delay(scope.scope_id)
```

Comment:
looks ok but you also want to inspect an api (from the scopes api) to sync a scope with id `SCOPE_ID` (so that if our github app is registered on a webhook, our backend can trigger OPAL to run a single repo sync)